baseapp-notifications/baseapp_notifications/utils.py

```diff
@@ -72,15 +75,13 @@ def send_notification(
             notification.save(update_fields=["emailed"])
 
     if send_push:
-        send_push_notification.delay(
+        kwargs.pop("level", None)
+        send_push_notification(
             recipient.id,
-            dict(
-                description=description,
-                verb=verb,
-                **kwargs
-                # TO DO:
-                # serialize all objects so devices can use this data if necessary
-            ),
+            push_title=push_title,
```

Comment:
I think `push_title` could be something like this:

```suggestion
            push_title=push_title or settings.MOBILE_APP_NAME,
```

I noticed this as pattern, when a title is not set it uses the APP_NAME... or iOS and Android already handle this when we sent an empty or None title?